README.md

```diff
@@ -1 +1,61 @@
-# chrony
\ No newline at end of file
+[![Build Status](https://travis-ci.org/oasis-roles/chrony.svg?branch=master)](https://travis-ci.org/oasis-roles/chrony)
+
+ROLE NAME
+===========
+
+Basic description for chrony
+
+Requirements
+------------
+
+Ansible 2.4 or higher
+
+Red Hat Enterprise Linux 7 or equivalent
+
+Valid Red Hat Subscriptions
+
+Role Variables
+--------------
+
+Currently the following variables are supported:
+
+### General
+
+* `chrony_package` - The package name to install on remote hosts (set to `chrony` by default)
+* `chrony_become_user` - The user to sign into when running commands on remote hosts (set to `root` by default)
+* `chrony_conf_template_path` - Path to a jinja2 template for generating the chrony configuration file (set to `chrony.conf` by default)
+* `chrony_conf_path` - Path in which to store the generated chrony configuration file on the remote hosts (set to `/etc/chrony.conf` by default)
+* `chrony_servers` - A list of servers and their options to be used with the chrony service (set to public servers from the pool.ntp.org project by default)
+* `chrony_driftfile_path` - Path in which to store files on the remote hosts that record the rate at which the systems' clocks gain/lose time (set to `/var/lib/chrony/drift` by default)
+* `chrony_makestep_args` - arguments to pass into the makestep option for chrony (set to `1.0 3` by default)
+* `chrony_enable_rtcsync` - Enable/disable kernel synchronization of the real-time clock (RTC) (set to `true` by default)
+* `chrony_hwtimestamp_interfaces` - A list of interfaces in which to enable hardware timestamping. Entered as a string separating each interface with a space (commented out by default)
```

Comment:
"commented out by default" is a bit confusing, to me. Commented out where? In the role? In my plays? My best guess is in `chrony.conf`. If that's the case, I would recommend saying something like, "Leave unset to use chrony defaults" for this and similar values in this list. For some, I would either give a brief sentence explaining what the Chrony default is or, better yet, link to Chrony's documentation on its default (thus future-proofing this document against changes in the default behavior of the service).

Reply:
What I meant by that is that the variables are commented out in defaults.yml and therefore not used by default. I will change the statement to "not configured by default" to be more clear